apps/frontend/app/(client)/(main)/course/_components/AssignmentCard.tsx

```diff
@@ -0,0 +1,69 @@
+'use client'
+
+import { AssignmentStatusTimeDiff } from '@/components/AssignmentStatusTimeDiff'
+import { cn, dateFormatter } from '@/libs/utils'
+import calendarIcon from '@/public/icons/calendar.svg'
+import type { Assignment } from '@/types/type'
+import Image from 'next/image'
+import 'react-circular-progressbar/dist/styles.css'
+import { StatusBadge } from '../../../(main)/_components/StatusBadge'
+
+const bgVariants = {
+  ongoing: 'bg-gradient-to-b from-blue-100 to-white',
+  upcoming: 'bg-white',
+  finished: 'bg-gray-500',
+  registeredOngoing: 'bg-gradient-to-b from-blue-100 to-white',
+  registeredUpcoming: 'bg-white'
+}
+
+const txtVariants = {
+  ongoing: 'text-blue-500',
+  upcoming: 'text-red-400',
+  finished: 'text-gray-500',
+  registeredOngoing: 'text-blue-500',
+  registeredUpcoming: 'text-red-400'
+}
+interface Props {
```

Comment:
```suggestion
interface AssignmentCardProps {
```

https://www.notion.so/skkuding/f825c8cdc9b74598bd00dadadf58815d?pvs=4#173ef9cff54580f48aabd956fd1bc0af

다른 곳도 마찬가지입니다~